lib/SearchAndSortPane/SearchAndSortPane.js

```diff
@@ -261,7 +270,7 @@ export class SearchAndSortPane extends Component {
             values={{ count: this.getSource().totalCount() }}
           />
         )}
-        lastMenu={(
+        lastMenu={hasLastMenu && (
```

Comment:
Of course,  renderHeader can be used for this purpose. But if there are some custom cases, so maybe,
it might be more useful to provide components for lastMenu (for example, an action menu with items) instead of hidding/showing.

Reply:
In case when it is not needed then I would need to provide something truthy. Imagine condition `lastMenu || (... existing code` in order to override it would be <div />. If you think this works fine I would change the implementation.

Reply:
As for me, the current solution looks very similar to` lastMenu || (... existing code`. 
Okay, we can go with the current solution in Q2.

Reply:
Then in my case in order to have it hidden, I would just provide `<div />`. If we both on the page it is not a problem for me to change it. Thank you for the comment.